Robert Smith, June 2024 VDOT changes

robsmi-msfte opened the pull request:
# June 2024 VDOT changes:

- Updated documentation in .PS1 to update last touched date, and a few minor verbiage changes
- Updated list of apps in '**AppxPackages.json**' with some new Windows 11 apps
- Changed a default user setting called '**IconsOnly**' from true to false, so that it does not apply by default.  This caused at least one icon (Teams) to display with a generic icon when accessed via Search.
- Removed final remnants of references to LGPO.exe, which was deprecated previously.  In doing this, changed "**LGPO**" event log source to "**LocalPolicy**" event log source, to align with how these policy settings are being applied.
- Fixed an issue where some settings were being applied only to local user and not default user profile.  Those settings were mirrored so they apply to both local profile of account being used to run VDOT, and default user profile.

raptorrico commented:
@robsmi-msfte - thanks for the new tag release! A request though... please could you be consistent with the tag names, latest adds a 'v' so 'v3.0.2009.0' vs the previous tags which don't have a leading 'v'. I download VDOT during a Packer build, extract and run. The path is now different. Thanks.

tmmuessig commented:
@raptorrico  - I just updated the release and removed the "v" from it.  Nothing else has changed.  Will keep this in mind as we move forward with new releases and thanks for the feedback.

robsmi-msfte commented:
I just put that 'v' in there because GitHub recommended it.  Thanks Tim for fixing that.

Thanks,

Robert M. Smith

From: Tim Muessig ***@***.***>
Sent: Wednesday, June 12, 2024 9:44 AM
To: The-Virtual-Desktop-Team/Virtual-Desktop-Optimization-Tool ***@***.***>
Cc: Robert Smith (AZURE) ***@***.***>; Mention ***@***.***>
Subject: Re: [The-Virtual-Desktop-Team/Virtual-Desktop-Optimization-Tool] Robert Smith, June 2024 VDOT changes (PR #213)


@raptorrico<https://github.com/raptorrico> - I just updated the release and removed the "v" from it. Nothing else has changed. Will keep this in mind as we move forward with new releases and thanks for the feedback.

-
Reply to this email directly, view it on GitHub<https://github.com/The-Virtual-Desktop-Team/Virtual-Desktop-Optimization-Tool/pull/213#issuecomment-2163049884>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AOX432KDR4U6M3OYTERKD5LZHBGD3AVCNFSM6AAAAABJEUJHVGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCNRTGA2DSOBYGQ>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>
